src/hitchhiker/tree/utils/clojure/async.cljc

```diff
@@ -0,0 +1,78 @@
+(ns hitchhiker.tree.utils.clojure.async
```

Comment:
Is the `clojure`-part really needed here?

Reply:
or is it supposed to be clj only?

Reply:
> Is the `clojure`-part really needed here?

Yes, this is due to how macros expand between the different environments.

Reply:
> or is it supposed to be clj only?

Still needs to be more deeply reviewed but is probably best kept this way for now. There is the possibility to autoalias clojure.* to exisiting cljs.* namespaces which is possible as of this commit to ClojureScript
https://github.com/clojure/clojurescript/commit/23632baa35f86de8866dede624545bc0cdf4a2bb